scripts/protocgen.sh

```diff
@@ -27,11 +27,11 @@ done
 
 # command to generate docs using protoc-gen-doc
 buf protoc \
--I "proto" \
--I "third_party/proto" \
---doc_out=./docs/core \
---doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
-$(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
+     -I "proto" \
+     -I "third_party/proto" \
+     --doc_out=./docs/core \
+     --doc_opt=./docs/protodoc-markdown.tmpl,proto-docs.md \
+     $(find "$(pwd)/proto" -maxdepth 5 -name '*.proto')
```

Comment:
Revert.

Reply:
Why? It's more readable to add and indent. In fact this was automatically formatted - I had to temporarily remove it when generating protobuf because docs didn't work.

Reply:
It is automatically formatted as per your personal taste or editor configuration. Furthermore, this change is out of scope of this PR. Please revert.

Reply:
BTW the whole file is formatted with 2-space tabs. Please **always adhere to all existing conventions before making changes**.